src/substrait/textplan/StructuredSymbolData.h

```diff
@@ -25,8 +27,7 @@ struct RelationData {
   std::vector<const ::substrait::proto::Rel*> newPipelines;
 
   // Column name for each field known to this relation (in field order).
-  // TODO -- Use symbols instead of column names here.
-  std::vector<std::string> fieldReferences;
+  std::vector<const SymbolInfo*> fieldReferences;
```

Comment:
Are these input fields or output fields?  If a relation has multiple inputs then how are these distributed?

Reply:
Added an indication of their current use (both input and output for now).